src/Map/src/Point.php

```diff
@@ -43,4 +43,16 @@ public function toArray(): array
             'lng' => $this->longitude,
         ];
     }
+
+    /**
+     * @param array{lat: float, lng: float}|array{0: float, 1: float} $point
+     */
+    public static function fromArray(array $point): self
+    {
+        if (isset($point['lat'], $point['lng'])) {
```

Comment:
```suggestion
        if ($point['lat'] ?? null && $point['lng'] ?? null) {
```

so we can allow users to pass `0` as lat/lng

Reply:
```php
$pos = ['lat' => 0, 'lng' => 0];

echo isset($pos['lat'], $pos['lng']);

# 1
```

The only case would be `$pos = [0, 0]` and i don't mind that much 😅